tests/acceptance/features/apiContract/propfind.feature

```diff
@@ -6,19 +6,39 @@ Feature: Propfind test
     Given these users have been created with default attributes and without skeleton files:
       | username |
       | Alice    |
+      | Brian    |
     And using spaces DAV path
     And the administrator has given "Alice" the role "Space Admin" using the settings api
     And user "Alice" has created a space "new-space" with the default quota using the GraphApi
 
 
-  Scenario: check the PROPFIND request of a space
+  Scenario: space-admin check the PROPFIND request of a space
     Given user "Alice" has uploaded a file inside space "new-space" with content "some content" to "testfile.txt"
     When user "Alice" sends PROPFIND request to space "new-space" using the WebDAV API
     Then the HTTP status code should be "207"
     And for user "Alice" the "PROPFIND" response should contain a space "new-space" with these key and value pairs:
-      | key                          | value            |
-      | oc:fileid                    | UUIDof:new-space |
-      | oc:name                      | new-space        |
-      | oc:permissions               | SRDNVCKZ         |
-      | oc:privatelink               |                  |
-      | oc:size                      | 12               |
+      | key            | value            |
+      | oc:fileid      | UUIDof:new-space |
+      | oc:name        | new-space        |
+      | oc:permissions | SRDNVCKZ         |
+      | oc:privatelink |                  |
+      | oc:size        | 12               |
+
+
+  Scenario Outline: space member with different role check the PROPFIND request of a space
+    Given user "Alice" has uploaded a file inside space "new-space" with content "some content" to "testfile.txt"
+    And user "Alice" has shared a space "new-space" to user "Brian" with role "<role>"
+    When user "Brian" sends PROPFIND request to space "new-space" using the WebDAV API
+    Then the HTTP status code should be "207"
+    And for user "Alice" the "PROPFIND" response should contain a space "new-space" with these key and value pairs:
```

Comment:
```suggestion
    And for user "Brian" the "PROPFIND" response should contain a space "new-space" with these key and value pairs:
```

I haven't looked at what the code does for this test step - why do we even need to mention the user name?

Reply:
The user name is used when checking the `oc:fileid` value - the code sends another PROPFIND to find out the file id, and checks that it is the same as in the PROPFIND that was done in the `When` step above.

We don't really care which user name is used here, we just need one that will be able to authenticate and have some access to the space. So the "problem" is just that the step looks a bit strange when it mentions Brian in the "when" step, the suddenly mentions Alice in this step.

Reply:
@phil-davis https://github.com/owncloud/ocis/pull/5266 Thanks for this PR for refactoring it. This PR shall wait for it.